### DISCUSSION TITLE
kafka集群并没有开启认证

### DISCUSSION THREAD

**Discussion by sumghost** [apache/doris#36394] [A - General / Q&A]
ErrorReason{code=errCode = 4, msg='errCode = 2, detailMessage = Failed to get all partitions of kafka topic: t-topic from backend[13402]. error: errCode = 2, detailMessage = failed to get kafka partition info: [(dksyb-104)[INTERNAL_ERROR]no partition in this topic] may be Kafka properties set in job is error or no partition in this topic that should check Kafka'}

**Comment by sumghost** [comment]
kafka routine doris

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
